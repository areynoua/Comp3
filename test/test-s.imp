begin
  read(a);
  print(a);
  b := 42;
  a := -5;
  b := a;
  b := (a + 8);
  b := (5 - a * (a / 9));
  if a < (b + 100) then
    a := 55
  else
    a := 44
  endif
end
