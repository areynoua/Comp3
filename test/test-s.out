-5
200
2
4
6
8
10
27
27
24
24
21
21
18
18
15
15
12
12
9
9
6
6
3
3
0
0
0
3
2
5
3
3
3
3
3
4
8
8
35
success
